game/templates/game/board.html

```
@@ -489,35 +372,26 @@
-                </div>
-                <div class="online-status">
-                    <span id="status-indicator"></span>
-                    <span id="status-text">Online</span>
-                </div>
 
-                <div id="streak-counter" class="streak-counter" style="display:none;">
-                    🔥 <span id="streak-count">0</span> Day Streak
+            <!-- Opponent Info (Top) -->
+            <div class="player-info top-player">
+                <div class="player-details">
+                    <img id="blackPlayerAvatar" class="nav-avatar" src="" style="display:none; width:20px; height:20px; border-radius:50%; object-fit:cover;">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)